op-e2e/system/gastoken/gastoken_test.go

```diff
@@ -39,10 +39,6 @@ func TestMain(m *testing.M) {
 	op_e2e.RunMain(m)
 }
 
-func TestCustomGasToken_L2OO(t *testing.T) {
-	testCustomGasToken(t, config.AllocTypeL2OO)
-}
-
 func TestCustomGasToken_Standard(t *testing.T) {
```

Comment:
We can actually delete this whole test, not part of this PR

Reply:
https://github.com/ethereum-optimism/optimism/pull/14094

Reply:
Done when rebasing